test/StaticValidatorTest.php

```diff
@@ -154,7 +154,6 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
 
     public function testExecuteValidWithParameters()
     {
-        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
```

Comment:
iirc, the behavior is, on StaticValidator::execute, the args is an array of parameter. at this case, args will be extracted. First parameter as min, and second as max. 

Warm regards,

Abdul Malik Ikhsan

Pada 8 Jul 2015, pukul 16.13, Alex Denvir <notifications@github.com> menulis:

> In test/StaticValidatorTest.php:
> 
> > @@ -154,7 +154,6 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
> >  
> >      public function testExecuteValidWithParameters()
> >      {
> > -        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
> Actually, I'm not sure it ever was. It definitely isn't valid with my changes at least.
> 
> As far as I can tell, passing this array in to the constructor for Between results in the following set as options (in the master branch):
> 
> protected $options = [
>     // default options in master branch
>     'inclusive' => true,
>     'min' => 0,
>     'max' => PHP_INT_MAX,
>     // options created by parent:__construct
>     0 => 1,
>     1 => 10
> ];
> 
> Because the class has default options (in master), no error is encountered in this test because 5 **is** between 0 and PHP_INT_MAX
> —
> Reply to this email directly or view it on GitHub.
>